From: "Jason St. John" <jstjohn@purdue.edu>
Date: Fri, 14 Feb 2014 22:07:22 -0500
Subject: man: use spaces instead of tabs

Several sections of the man pages included intermixed tabs and spaces;
this commit replaces all tabs with spaces.

(cherry picked from commit b200a92cdc47b204e2ed7fbdd1eaf15abfd019c6)

Conflicts:
	man/crypttab.xml
	man/sd_bus_creds_get_pid.xml
	man/systemd.service.xml
---
 man/systemd-delta.xml   | 68 ++++++++++++++++++++++++-------------------------
 man/systemd.service.xml |  4 +--
 man/systemd.socket.xml  | 20 +++++++--------
 3 files changed, 46 insertions(+), 46 deletions(-)

diff --git a/man/systemd-delta.xml b/man/systemd-delta.xml
index f802aa5..23fb8d1 100644
--- a/man/systemd-delta.xml
+++ b/man/systemd-delta.xml
@@ -50,8 +50,8 @@
         <refsynopsisdiv>
                 <cmdsynopsis>
                         <command>systemd-delta</command>
-			<arg choice="opt" rep="repeat">OPTIONS</arg>
-			<arg choice="opt" rep="repeat"><replaceable>PREFIX</replaceable><optional>/<replaceable>SUFFIX</replaceable></optional>|<replaceable>SUFFIX</replaceable></arg>
+                        <arg choice="opt" rep="repeat">OPTIONS</arg>
+                        <arg choice="opt" rep="repeat"><replaceable>PREFIX</replaceable><optional>/<replaceable>SUFFIX</replaceable></optional>|<replaceable>SUFFIX</replaceable></arg>
                 </cmdsynopsis>
         </refsynopsisdiv>
 
@@ -78,27 +78,27 @@
                 the name of the main configuration file, must match).
                 For a fuller explanation, see
                 <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
-		</para>
-
-		<para>The command line argument will be split into a
-		prefix and a suffix. Either is optional. The prefix
-		must be one of the directories containing
-		configuration files (<filename>/etc</filename>,
-		<filename>/run</filename>,
-		<filename>/usr/lib</filename>, ...). If it is given,
-		only overriding files contained in this directory will
-		be shown. Otherwise, all overriding files will be
-		shown. The suffix must be a name of a subdirectory
-		containing configuration files like
-		<filename>tmpfiles.d</filename>,
-		<filename>sysctl.d</filename> or
-		<filename>systemd/system</filename>. If it is given,
-		only configuration files in this subdirectory (across
-		all configuration paths) will be analyzed. Otherwise,
-		all configuration files will be analyzed. If the
-		commandline argument is not given at all, all
-		configuration files will be analyzed. See below for
-		some examples.</para>
+                </para>
+
+                <para>The command line argument will be split into a
+                prefix and a suffix. Either is optional. The prefix
+                must be one of the directories containing
+                configuration files (<filename>/etc</filename>,
+                <filename>/run</filename>,
+                <filename>/usr/lib</filename>, ...). If it is given,
+                only overriding files contained in this directory will
+                be shown. Otherwise, all overriding files will be
+                shown. The suffix must be a name of a subdirectory
+                containing configuration files like
+                <filename>tmpfiles.d</filename>,
+                <filename>sysctl.d</filename> or
+                <filename>systemd/system</filename>. If it is given,
+                only configuration files in this subdirectory (across
+                all configuration paths) will be analyzed. Otherwise,
+                all configuration files will be analyzed. If the
+                commandline argument is not given at all, all
+                configuration files will be analyzed. See below for
+                some examples.</para>
         </refsect1>
 
         <refsect1>
@@ -200,21 +200,21 @@
                 </variablelist>
         </refsect1>
 
-	<refsect1>
-		<title>Examples</title>
+        <refsect1>
+                <title>Examples</title>
 
-		<para>To see all local configuration:</para>
-		<programlisting>systemd-delta</programlisting>
+                <para>To see all local configuration:</para>
+                <programlisting>systemd-delta</programlisting>
 
-		<para>To see all runtime configuration:</para>
-		<programlisting>systemd-delta /run</programlisting>
+                <para>To see all runtime configuration:</para>
+                <programlisting>systemd-delta /run</programlisting>
 
-		<para>To see all system unit configuration changes:</para>
-		<programlisting>systemd-delta systemd/system</programlisting>
+                <para>To see all system unit configuration changes:</para>
+                <programlisting>systemd-delta systemd/system</programlisting>
 
-		<para>To see all runtime "drop-in" changes for system units:</para>
-		<programlisting>systemd-delta --type=extended /run/systemd/system</programlisting>
-	</refsect1>
+                <para>To see all runtime "drop-in" changes for system units:</para>
+                <programlisting>systemd-delta --type=extended /run/systemd/system</programlisting>
+        </refsect1>
 
         <refsect1>
                 <title>Exit status</title>
diff --git a/man/systemd.service.xml b/man/systemd.service.xml
index c116a9b..0f4be71 100644
--- a/man/systemd.service.xml
+++ b/man/systemd.service.xml
@@ -769,8 +769,8 @@ ExecStart=/bin/echo $ONE $TWO ${TWO}</programlisting>
 				exit statuses is merged. If the empty
 				string is assigned to this option, the
 				list is reset, all prior assignments
-				of this option will have no
-				effect.</para></listitem>
+                                of this option will have no
+                                effect.</para></listitem>
                         </varlistentry>
 
                         <varlistentry>
diff --git a/man/systemd.socket.xml b/man/systemd.socket.xml
index da0f2aa..39fba51 100644
--- a/man/systemd.socket.xml
+++ b/man/systemd.socket.xml
@@ -121,16 +121,16 @@
                 boot or late system shutdown should disable this
                 option.</para>
 
-		<para>Socket units will have a
-		<varname>Before=</varname> dependency on the service
-		which they trigger added implicitly. No implicit
-		<varname>WantedBy=</varname> or
-		<varname>RequiredBy=</varname> dependency from the
-		socket to the service is added. This means that the
-		service may be started without the socket, in which
-		case it must be able to open sockets by itself. To
-		prevent this, an explicit <varname>Requires=</varname>
-		dependency may be added.</para>
+                <para>Socket units will have a
+                <varname>Before=</varname> dependency on the service
+                which they trigger added implicitly. No implicit
+                <varname>WantedBy=</varname> or
+                <varname>RequiredBy=</varname> dependency from the
+                socket to the service is added. This means that the
+                service may be started without the socket, in which
+                case it must be able to open sockets by itself. To
+                prevent this, an explicit <varname>Requires=</varname>
+                dependency may be added.</para>
 
                 <para>Socket units may be used to implement on-demand
                 starting of services, as well as parallelized starting
